conf/ethz_euler.config

```diff
@@ -0,0 +1,51 @@
+/*
+* ------------------------------------------------------------------
+*   nf-core pipelines config file for the ETH Zurich Euler cluster
+* ------------------------------------------------------------------
+*/
+
+params {
+        config_profile_name        = 'ethz_euler'
+        config_profile_description = 'ETH Zurich Euler cluster profile -- provided by nf-core/configs'
+        config_profile_contact     = 'Joao Agostinho de Sousa (github: @jpadesousa; email: joao.agostinhodesousa@hest.ethz.ch)'
+        config_profile_url         = 'https://scicomp.ethz.ch/wiki/Euler'
+
+        max_memory = 4.TB
+        max_cpus   = 128
+        max_time   = 120.h
+        
+	igenomes_base   = '/cluster/project/igenomes'
+        igenomes_ignore = false
+}
+
+executor {
+        name                = 'slurm'
+        perCpuMemAllocation = true
+}
+
+process {
+        scratch             = 'true'
+        errorStrategy       = 'retry'
+        maxRetries          = 3
+        maxErrors           = '-1'
```

Comment:
What is the intention of this? https://www.nextflow.io/docs/latest/process.html#maxerrors doesn't document strings or -1 (but says disabled by default).

Reply:
Hi @pontus,
Yes, you are right. That can be removed.